e1039-analysis/SimHits/setup_mye1039.sh

```diff
@@ -1,4 +1,4 @@
-export MY_E1039=/seaquest/users/yfeng/mye1039/e1039-core/this-e1039.sh
+export MY_E1039=/seaquest/users/$USER/core-inst/this-e1039.sh
```

Comment:
the default path in spinquest cluster is `/e906/app/software/osg/software/e1039/this-e1039.sh`. should we keep that path or use this one?

Reply:
ah so I think that path is already in `setup.sh` but this setup script would setup the e1039-core in each user's installation - I added a instruction in the readme to request each user to change the path here if the installation is somewhere else